src/Game.js

```diff
@@ -0,0 +1,213 @@
+import React, { Component } from "react";
+import Canvas from "./Canvas";
+
+class Game extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      player1: { name: null, id: null, paddle: [235, 480] },
```

Comment:
initial paddle positions should be captured in a variable somewhere, like a `Player` class. rather than inject the paddle location, inject whether they are player 1 or player 2. then in the constructor, depending on which player they are, assign their paddle the appropriate coordinates.